### PR TITLE
PH for Norway

### PR DESCRIPTION
**Pull request by Gazer75**

### CODE REVIEW

File: holidays/no.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+_nominatim_url: 'https://nominatim.openstreetmap.org/reverse?format=json&lat=61.0&lon=8.0&zoom=8&addressdetails=1&accept-language=no,en'
+
+PH:  # https://en.wikipedia.org/wiki/Public_holidays_in_Norway
+  'Nyttårsdag': [1, 1]
+  'Skjærtorsdag': [easter, -3]
+  'Langfredag': [easter, -2]
+  'Påskedag': [easter, 0]
+  '2. Påskedag': [easter, 1]
+  '1. Mai': [1, 5]
+  'Grunnlovsdagen': [17, 5]
```
Comment (ypid) [inline holidays/no.yaml]: The numbers should be flipped, first month then day ;-)

Comment (Gazer75) [inline holidays/no.yaml]: First time I've tried this on Github so not sure where to actually make changes now.
I've only created or commented on issues in here before :)

**Comment by Gazer75** [comment]
First time ever doing this so not sure how things work :)

**Comment by ypid** [comment]
Thanks! You are starting to get the hang of it :wink: Note that I fixed a second instance of swapped month/days in https://github.com/opening-hours/opening_hours.js/commit/94ee7734077c29fd3cb6351cab8d39cdc6b7f03a before I merged. Changes should be live :)

Nerd talk ahead: The CI tests are not passing because this branch was not rebased ontop of #229 which I opened after I saw that CI fails in this PR.

**Pull request merged**